### PR TITLE
Fix cross-repo link

### PR DESCRIPTION
**Pull request by sttts**
Fixing https://github.com/kubernetes/kubernetes/issues/47211

### CODE REVIEW

**Comment by sttts** [comment]
Not sure there is a better solution. The link must work inside of staging and in the client-go repo. Making in-project+cross-project links seems to be impossible.

**Comment by ahmetb** [comment]
@sttts still 404 for me: https://github.com/kubernetes/apiextensions-apiserver/ Is this repo private?

**Comment by sttts** [comment]
@ahmetb it doesn't exist yet ;-) Compare by kubernetes-dev mail from yesterday.

File: staging/src/k8s.io/client-go/examples/custom-resources/README.md [hunk 1]
```diff
@@ -4,4 +4,4 @@
 
 For a client-go example using CustomResourceDefinitions, go to
 
-  http://github.com/kubernetes/apiextensions-apiserver/examples/client-go.
+  [k8s.io/apiextensions-apiserver/examples/client-go](https://github.com/kubernetes/apiextensions-apiserver/tree/master/examples/client-go).
```
Comment (cblecker) [inline staging/src/k8s.io/client-go/examples/custom-resources/README.md]: @thockin created https://git.k8s.io for stuff like this.
Try https://git.k8s.io/apiextentions-apiserver/examples/client-go

**Review by cblecker — CHANGES_REQUESTED** [review]

**Comment by ahmetb** [comment]
This directory probably should not have been merged in the first place if it doesn't exist yet? 😕

File: staging/src/k8s.io/client-go/examples/custom-resources/README.md (hunk 1 above)
Comment (thockin) [inline staging/src/k8s.io/client-go/examples/custom-resources/README.md]: that's a dead link right now...

Comment (sttts) [inline staging/src/k8s.io/client-go/examples/custom-resources/README.md]: done

**Review by cblecker — APPROVED** [review]
This looks good to me, except that `kubernetes/apiextensions-apiserver` is empty. Once there is content here, then this can be merged.

**Comment by sttts** [comment]
Repo exists, but is still empty. @caesarxuchao is updating the bot now.

**Comment by ahmetb** [comment]
pull-kubernetes-federation-e2e-gce is flaky: #47672

**Comment by spiffxp** [comment]
OK, there's content here https://github.com/kubernetes/apiextensions-apiserver
Going to lgtm based on @cblecker's review
/assign

File: staging/src/k8s.io/client-go/examples/custom-resources/README.md [hunk 2]
```diff
@@ -4,4 +4,4 @@
 
 For a client-go example using CustomResourceDefinitions, go to
 
-  http://github.com/kubernetes/apiextensions-apiserver/examples/client-go.
+  [k8s.io/apiextensions-apiserver/examples/client-go](https://git.k8s.io/apiextentions-apiserver/examples/client-go).
```
Comment (nikhita) [inline staging/src/k8s.io/client-go/examples/custom-resources/README.md]: s/apiextentions/apiextensions. fix in #47684

**Pull request merged**